firefox-ios/Client/Coordinators/Router/RouteBuilder.swift

```diff
@@ -156,9 +156,19 @@ final class RouteBuilder {
             return .search(url: nil, isPrivate: false)
         }
 
+        var isBrowsingActivity: Bool {
+            if featureFlags.isFeatureEnabled(.universalLinks, checking: .buildOnly) {
+                return userActivity.activityType == NSUserActivityTypeBrowsingWeb ||
+                userActivity.activityType == browsingActivityType
+            } else {
+                return true
+            }
+        }
```

Comment:
This would be nice to be made a class var something like below so you can use it anywhere in the router :) 

https://github.com/mozilla-mobile/firefox-ios/blob/main/firefox-ios/Client/Coordinators/EnhancedTrackingProtectionCoordinator.swift#L32-L34

Reply:
Piggybacking on this comment, in future if you like you can add a setting option too to enable this feature using our debug menu.

Reply:
sure. It wasn't being used anywhere else so I isolated it to the method but I can deff move it out

Reply:
actually. This is the only part of the router that is even remotely related to user activities. Is it really worth moving this out?

Reply:
Just seems cleaner to put it on top of class so your makeRoute method becomes less cluttered without computed var but then this is also a slightly odd scenario where your computed var is used below ah! 
Ideally I would suggest making a guard but you can't really do that here (maybe you can) anyways go with your best judgment here :)

Reply:
fair enough. I moved it to a private func since we need to take in an input of user activity